components/automate-ui/src/app/components/pending-edits-bar/pending-edits-bar.component.spec.ts

```diff
@@ -219,6 +171,156 @@ describe('PendingEditsBarComponent', () => {
   });
 });
 
+describe('PendingEditsBarComponent--unauthorized', () => {
+  let component: PendingEditsBarComponent;
+  let fixture: ComponentFixture<PendingEditsBarComponent>;
+  let store: Store<NgrxStateAtom>;
+
+  beforeEach(async(() => {
+    configureWith({
+      '/iam/v2/projects': genPerm('/iam/v2/projects', false)
+    });
```

Comment:
This new set of tests is for the unauthorized case. Notice the TestBed is configured with `false` here. Then the exact same set of tests earlier in the file (not shown in the diff here because they were not technically not changed), which caused the <kbd>edits pending</kbd> bar to be visible, will now show that the bar is hidden.